dashboard/test/controllers/pd/workshop_dashboard_controller_test.rb

```diff
@@ -1,21 +1,21 @@
 require 'test_helper'
 
 class Pd::WorkshopDashboardControllerTest < ::ActionController::TestCase
-  test 'admins can access the dashboard' do
+  test 'admins can access the dashboard and get correct permission value' do
```

Comment:
Good suggestion. I thought initially providing a block could get cumbersome, but once we remove the array capability then it might not be too bad. Of course we'd want to encourage not asserting too many things in a single test case. It could look something like this (after Dave's suggested rename):

```ruby
test_user_gets_response_for :index, user: :admin do
  assert_equal :admin, assigns(:permission)
end
```